version/1/4/th/code_of_conduct.md

```diff
@@ -0,0 +1,50 @@
+# จรรยาบรรณฉบับของกติกามารยาท 
+
+## คำมั่นสัญญาของเรา
+
+เพื่อส่งเสริมสภาพแวดล้อมที่เปิดกว้างและเป็นที่ยอมรับ 
+เราในฐานะผู้ร่วมให้การสนับสนุนและผู้ดูแล สัญญาว่าในโครงการและชุมชนของเราจะเป็นประสบการณ์ ที่ปราศจากอคติสำหรับทุกคน โดยไม่คำนึงถึง อายุ ขนาดของร่างกาย ความพิการ ชาติพันธุ์, เอกลักษณ์ทางเพศและการแสดงออก, ระดับประสบการณ์,สัญชาติ เชื้อชาติ และ ศาสนา
+
+## มาตรฐานของเรา
+
+ตัวอย่างของพฤติกรรมที่ก่อให้เกิดสภาพแวดล้อมที่ดี
+* ใช้ภาษาที่แสดงถึงการทำงานเป็นทีม ที่กระตุ้นให้เกิดสิ่งแวดล้อมที่ดี
+* เคารพในมุมมองและประสบการณ์ที่ต่างกัน
+* ยอมรับอย่างสุภาพในเชิงวิจารณ์ที่สร้างสรรค์
+* มุ่งเน้นที่สิ่งที่ดีที่สุดสำหรับชุมชน
+* แสดงความเห็นอกเห็นใจต่อสมาชิกชุมชนคนอื่น ๆ
+
+ตัวอย่างของพฤติกรรมที่ไม่สามารถยอมรับได้โดยผู้เข้าอบรม
+* การใช้ภาษาหรือภาพ ที่ส่อถึงการคุกคามทางเพศ 
+*การแสดงความคิดเห็น เพื่อการโจมตีส่วนบุคคลหรือทางการเมือง โดยไม่คำนึงถึงเหตุและผล
+* การล่วงละเมิดในที่สาธารณะหรือส่วนตัว
+* เผยแพร่ข้อมูลส่วนตัวของผู้อื่นเช่น ที่อยู่ อีเมล์ เฟสบุ๊ค โดยไม่ได้รับอนุญาต
```

Comment:
I don't think เฟสบุ๊ค is suitable in this context.
E-mail in Thai is อีเมล not อีเมล์